website/docs/privacy.md

```diff
@@ -1,22 +0,0 @@
----
-title: Privacy Policy
-keywords: [ hudi, privacy]
-last_modified_at: 2019-12-30T15:59:57-04:00
----
-
-Information about your use of this website is collected using server access logs and a tracking cookie.
-The collected information consists of the following:
-
-* The IP address from which you access the website;
-* The type of browser and operating system you use to access our site;
-* The date and time you access our site;
-* The pages you visit;
-* The addresses of pages from where you followed a link to our site.
-
-Part of this information is gathered using a tracking cookie set by the [Google Analytics](http://www.google.com/analytics) service and handled by Google as described in their [privacy policy](http://www.google.com/privacy). See your browser documentation for instructions on how to disable the cookie if you prefer not to share this data with Google.
-
-We use the gathered information to help us make our site more useful to visitors and to better understand how and when our site is used. We do not track or collect personally identifiable information or associate gathered data with any personally identifying information from other sources.
-
-By using this website, you consent to the collection of this data in the manner and for the purpose described above.
-
-The Hudi development community welcomes your questions or comments regarding this Privacy Policy. Send them to dev@hudi.apache.org
```

Comment:
we have a footer item "Privacy" which links to an asf common page for privacy. so let's make that footer item go to this page, then add the asf common page to this page as a link called "ASF Data Privacy". so we keep this original content.